### DISCUSSION TITLE
Возможность создания одного большого pdf/cbz.

### DISCUSSION THREAD

**Discussion by Semeneater** [lirix360/ReadmangaGrabber#38] [Ideas]
Местами не хватает опции, чтобы все скачанные глава объединялись в один файл. Было бы очень круто!

**Comment by ghost** [comment]
> Местами не хватает опции, чтобы все скачанные глава объединялись в один файл. Было бы очень круто!

[ilovepdf](https://www.ilovepdf.com/ru/merge_pdf) тебе в помощь

**Comment by lirix360** [comment]
В версии 2.0.2 добавлена возможность создания PDF по томам.
